lmi-multiple-types.cc

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
```

Comment:
This example looks like lmi.cc, but just with the character array & pointer. Maybe move the content to lmi.cc?

Reply:
@ktran Since debug-stories are used for bug reproduction (and maybe demos), could it happen that changing lmi.cc renders some bug (or demo) instructions untrue? For instance, "set a break point on line x". 

On the other hand, broken instructions can be fixed as they are discovered. What do you think?

Reply:
Merged the two files into lmi.cc on @ktran's suggestion.